### ISSUE TITLE
Change the API for getting servers

### ISSUE THREAD

**Issue by asadboi123** [PlexRipper/PlexRipper#108]
Currently you're using "private const string _plexServerUrl = "https://plex.tv/pms/servers.xml";" but this doesn't show all servers!

You can use api/v2/ to get servers instead

https://plex.tv/api/v2/resources?X-Plex-Token=xxxxxxxxxxxxxx&X-Plex-Client-Identifier=chrome

**Comment by fxsth** [comment]
Seems to be the cause for #104

**Comment by JasonLandbridge** [comment]
Ah thank you very much for this, I will fix this is the next release (0.9.1) which should be available by the end of this week. Apologies for the delay!

**Comment by JasonLandbridge** [comment]
I can confirm this new api URL provides more servers and connections to communicate with the Plex server. So I'm currently working on integrating this. I also expect this to resolve the proxy issues so I will provide a way to select which connection to use but this is quite a bit of work. But I am working on it as we speak :+1:

**Comment by JasonLandbridge** [comment]
Update: It's been quite a bit of work but I hope to have it finished by the end of the week. The biggest change is that PlexRipper now receives multiple connections for each Plex server, you will be able to choose a preferred connection and each connection is also automatically tested if it actually works. This change had quite a ripple effect which is why it's taking longer. This will also solve the proxy connection issues and now you can connect to your own server through the local network, which is generally faster than connecting through a public ip

For real-time progress see: [ServerConnections Branch](https://github.com/PlexRipper/PlexRipper/commits/ServerConnections)

Like and subscribe to this issue to stay up to date :+1:

**Comment by turnerd10** [comment]
Seem to be running into the same issue as many others. Any update?

**Comment by JasonLandbridge** [comment]
Yes! Finally I have resolved this issue, among many others, which is now released [v0.10.0-RC1](https://github.com/PlexRipper/PlexRipper/releases/tag/v0.10.0-RC1) on both the latest and dev Docker channels:  https://hub.docker.com/r/plexripper/plexripper/tags
